doc/content/source/functions/BaselineCorrection.md

```diff
@@ -0,0 +1,409 @@
+
+!listing baselinecorrection/full_sine.i
+         block=Functions
+         id=demo-3-input
+         caption=Sample input syntax for applying the `BaselineCorrection` object to a full-sine acceleration time history.
+
+Notice that the polynomial order used for the correction of a full-sine wave, shown in [demo-3-input], is higher than that used for the correction of a half-sine wave, shown in [example-2] or [demo-2-input]. MASTODON's `BaselineCorrection` object produced a corrected acceleration that corresponds with the displacement time history shown in [demo-3-adjusted].
+
+!plot scatter filename=baselinecorrection_data/demo_3_adjusted.csv
+              data=[{'x':'time', 'y':'displacement', 'name':'Corrected Displacement'}]
+              layout={'xaxis':{'title':'Time'},
+                      'yaxis':{'title':'Displacement'},
+                      'title':'Corrected Displacement Time History'}
+              id=demo-3-adjusted
+              caption=Plot of the displacement time history resulting from integration of the baseline corrected values of the acceleration shown in [demo-1] extrapolated out to $t = 2$.
+
+### Correction of a Twenty-sine id=demo-4
+
+Consider the half-sine acceleration whose data points were provided as input for the `BaselineCorrection` object in [example-1] and which is plotted in [demo-4]. This acceleration can be defined, analytically, by the following function:
+
+!equation id=twenty-sine
+\ddot{u}(t) = -250 \pi^{2} sin(50 \pi t), \, \, \, \forall \, t\in[0, 0.8]
+
+!plot scatter filename=baselinecorrection/twenty_sines_accel.csv
+              data=[{'x':'time', 'y':'acceleration', 'name':'Nominal Acceleration'}]
+              layout={'xaxis':{'title':'Time'}, 'yaxis':{'title':'Acceleration'}, 'title':'Nominal Acceleration Time History'}
+              id=demo-4
+              caption=Plot of a twenty-cycle sinusoidal acceleration time history to be adjusted by the `BaselineCorrection` object.
+
+When the nominal acceleration is of high order and uniformity, a stronger correction is required. Thus, for the nominal time history given by [twenty-sine], combination +AVD+ was used with high-order polynomials for each type. MASTODON's `BaselineCorrection` object produced a corrected acceleration that corresponds with the displacement time history shown in [demo-4-adjusted].
+
+!plot scatter filename=baselinecorrection_data/demo_4_adjusted.csv
+              data=[{'x':'time', 'y':'displacement', 'name':'Corrected Displacement'}]
+              layout={'xaxis':{'title':'Time'},
+                      'yaxis':{'title':'Displacement'},
+                      'title':'Corrected Displacement Time History'}
+              id=demo-4-adjusted
+              caption=Plot of the displacement time history resulting from integration of the baseline corrected values of the acceleration shown in [demo-4].
+
+### Apply a Corrected Twenty-sine as a Boundary Condition id=demo-5
```

Comment:
If you consolidate `halfsine, fullsine, halfsinewithBC, twentysines, twentysineswithBC` to one, then consider modifying the this markdown file as well.

Reply:
@somu15 Ill just reference the correct block in all of the `!listing` calls to the single, comprehensive input file. 

But, I should keep the input file that uses `PresetAcceleration` separate I think.